cli/modes/Client.cs

```diff
@@ -44,6 +48,32 @@ class ClientMode : Mode
     private void SecureConnection() {
         connection.AttemptSecuringConnection();
     }
+    
+    private void FileDirectory(ArraySegment<string> opts) {
+        if (opts.Count > 1) {
+            Console.WriteLine("Invalid number of arguments!");
+            return;
+        }
+        if (opts.Count == 0)
+        {
+            connection.GetFileDirectory("");
+        }
+        if (opts.Count == 1)
+        {
+            connection.GetFileDirectory(opts[0]);
+        }
+    }
+    
+    private void ChangeDirectory(ArraySegment<string> opts) {
+        if (opts.Count > 1) {
+            Console.WriteLine("Invalid number of arguments!");
+            return;
+        }
+        if (opts.Count == 1)
+        {
+            connection.GetChangedDirectory(opts[0]);
```

Comment:
It shouldn't call a different function on Connection, Client should have a directory and all cd should do is change that value.